tree.go

```diff
@@ -267,5 +268,10 @@ var (
 	jxr     = newMIME("image/jxr", ".jxr", magic.Jxr).alias("image/vnd.ms-photo")
 	parquet = newMIME("application/vnd.apache.parquet", ".parquet", magic.Par1).
 		alias("application/x-parquet")
-	cbor = newMIME("application/cbor", ".cbor", magic.CBOR)
+	cbor   = newMIME("application/cbor", ".cbor", magic.CBOR)
+	netpbm = newMIME("image/x-portable-bitmap", ".pbm", magic.NetPBM)
+	netpgm = newMIME("image/x-portable-graymap", ".pgm", magic.NetPGM)
+	netppm = newMIME("image/x-portable-pixmap", ".ppm", magic.NetPPM)
+	netpfm = newMIME("image/x-portable-floatmap", ".pfm", magic.NetPFM)
```

Comment:
Linux `file` utility is most comprehensive file type identification and it does not support PFM. I think we're fine if we don't too.